src/BuildScriptGenerator/Node/NodeBashBuildSnippetProperties.cs

```diff
@@ -12,6 +12,8 @@ public class NodeBashBuildSnippetProperties
     {
         public string PackageRegistryUrl { get; set; }
 
+        public string PackageDirectory { get; set; }
```

Comment:
nit: may be NonRootPackageDirectory? can you add a test where packagedirectory is the root directory